### PR TITLE
Minor update

### PR DESCRIPTION
**Pull request by ThomasLandauer**
Added note to indicate that `$this->isGranted()` only works in a controller, not in an Authenticator.

### CODE REVIEW

File: security.rst [hunk 1]
```diff
@@ -910,7 +910,7 @@ Users with the ``ROLE_ADMIN`` role will also have the
 ``ROLE_USER`` role. And users with ``ROLE_SUPER_ADMIN``, will automatically have
 ``ROLE_ADMIN``, ``ROLE_ALLOWED_TO_SWITCH`` and ``ROLE_USER`` (inherited from ``ROLE_ADMIN``).
 
-For role hierarchy to work, do not try to call ``$user->getRoles()`` manually::
+For role hierarchy to work, do not try to call ``$user->getRoles()`` manually in a controller::
```
Comment (noniagriconomie) [inline security.rst]: in your code in general?

Comment (noniagriconomie) [reply security.rst]: sorry, 

I mean : 

> For role hierarchy to work, do not try to call ``$user->getRoles()`` manually in a controller or any other part of your code, prefer using the role hierarchy service

thing like that

**Comment by javiereguiluz** [comment]
Thanks Thomas!

File: security.rst (hunk 1 above)
Comment (xabbuh) [reply security.rst]: You mean calling `isGranted()` did not work because the method is not defined in the authenticator context?

Comment (ThomasLandauer) [reply security.rst]: Yes. Cause I would have assumed it works there.

**Pull request closed without merging**